lib/ransack/helpers/form_helper.rb

```diff
@@ -145,7 +145,7 @@ def html_options(args)
         private
 
           def parameters_hash(params)
-            if ::ActiveRecord::VERSION::MAJOR == 5 && params.respond_to?(:to_unsafe_h)
+            if (5..6).include?(::ActiveRecord::VERSION::MAJOR) && params.respond_to?(:to_unsafe_h)
```

Comment:
I would prefer this condition:

```suggestion
            if ::ActiveRecord::VERSION::MAJOR >= 5 && params.respond_to?(:to_unsafe_h)
```

Reply:
Done